test-resources/cljs/run-phantom.js

```diff
@@ -0,0 +1,52 @@
+var system = require('system');
```

Comment:
I'm not too familiar with the recommend testing approach for clojurescript, but have you considered using cljs.test and possible something like https://github.com/bensu/doo to run the tests?